### DISCUSSION TITLE
Skin

### DISCUSSION THREAD

**Discussion by Eaielectronic** [TurboWarp/extensions#1681] [Q&A]
Hello, I would like to know if it is possible to switch to a costume from another sprite with an extension to limit the number of images.

**Comment by SharkPool-SP** [comment]
> Hello, I would like to know if it is possible to switch to a costume from another sprite with an extension to limit the number of images.

Skins extension + Sharktilities (my gallery)

Sharktilities has a block similar to Looks+ that can get costumes of other sprites

**Comment by Eaielectronic** [comment]
Thanks
